ref: `remote modify` `gdrive_acknowledge_abuse` and `profile` (#3795)
iterative/dvc.org: content/docs/contributing/core.md

> the Google account.

To avoid tests flow interruption by manual login, perform authorization once and

> DVC will automatically cache the obtained Google Drive access token in a global
> file (e.g. by default `~/.cache/pydrive2fs/$client_id/default.json` on Linux) to
> avoid manual login next time.

Reviewer: ```suggestion
DVC will automatically cache the Google Drive access token obtained in a global
location (e.g. `~/.cache/pydrive2fs` on Linux, see [details]) to avoid manual
logins after this.

[details]: https://dvc.org/doc/command-reference/remote/modify#google-drive
```

Reviewer: Would be ideal to move the `[details]` ref link to the bottom of the section.
